src/site/content/en/blog/web-animations/index.md

```diff
@@ -20,12 +20,12 @@ When used correctly, [animations improve user perception and memory](https://www
 
 The [Web Animations API](https://www.w3.org/TR/web-animations-1/) is a tool that enables developers to write [imperative animations with JavaScript](https://www.youtube.com/watch?v=WaNoqBAp8NI). It was written to underpin both CSS animation and transition implementations and enable future effects to be developed, as well as existing effects to be composed and timed.
 
-While it has a long history, originally launching in Chromium 36, the latest release, Chromium 84, brings us a slew of previously unsupported features. 
+While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), have already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in their browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```

Comment:
```suggestion
While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), has already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in its browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```

Reply:
The Google standard is American English. I don't know why I didn't request this change before.